### ISSUE TITLE
Support mode without the source buffer

### ISSUE THREAD

**Issue by odnoletkov** [bfrg/vim-jqplay#2]
Quite often I use jq without the source file: `jq -n <filter>`. For example to generate some sample data or as a scratch pad for some calculation.

It would be great if vim-jqplay could support that mode. For example the `Jqplay` command could be enabled globally – when run in non-json button it would create just the filter-output buffers pair.

Currently to achieve something similar multiple commands are needed
```
:new
:setf json
:Jqplay -n
:hide
```

**Comment by bfrg** [comment]
I have never used the `-n` option myself. There's no need to open an input buffer if it's not used by jq. This can be improved.

But in general, if you want to start jqplay from anywhere, take a look at the 3rd [example](https://github.com/bfrg/vim-jqplay/tree/413ada343fa3f07047e28d44b6a0c903f0d4614e#example-3-jqplayscratch) (or under `:help jqplay-faq`). 

You can modify the command so that the session always opens in a new tab page:
```vim
" in vimrc
command! -nargs=? -complete=customlist,jqplay#complete JqplayScratch tab new |
        \ setlocal buflisted buftype=nofile bufhidden=hide noswapfile filetype=json |
        \ <mods> Jqplay <args>
```

Like I said in #1, I can add the `:JqplayScratch` command to the plugin. Seems like it should be built-in.

**Comment by odnoletkov** [comment]
I think the ideal solution is `:JqplayScratch` command moving to the plugin *and* supporting the `-n`/`--null-input` argument – this will create an input-less session with only filter and output buffers.

**Comment by bfrg** [comment]
You can try out the [null-input-support](https://github.com/bfrg/vim-jqplay/tree/null-input-support) branch.

**Comment by odnoletkov** [comment]
This is exactly what I wanted – thank you so much!

**Comment by bfrg** [comment]
Fixed by #4.
